SpringProject/src/main/java/com/gradproject2019/userConference/controller/UserConferenceController.java

```diff
@@ -27,4 +25,10 @@
         UserConferenceResponseDto newInterest = userConferenceService.saveInterest(token, userConferenceRequestDto);
         return ResponseEntity.ok(newInterest);
     }
+
+    @GetMapping(path = "/user-conferences/{id}")
```

Comment:
So there is a slight issue with this - without documentation any FE wouldn't know what the ID is referring to. Is it the Conference or the User? 
As you're developing the FE/BE so closely together and its working for you don't change it, but a path like `user/{id}/conferences` would make more sense to me that this returns a list of conferences associated to a user

Reply:
Ah I see what you are saying, I will discuss with FE and see if they would like me to rewrite this